ateam_vision_filter/src/types/robot_measurement.hpp

```diff
@@ -27,6 +27,34 @@ struct RobotMeasurement
 {
   Eigen::Vector2d position;
   double theta;
+
+  void invert()
+  {
+    position *= -1.0;
+    theta = angle_wrap_pi(theta + M_PI);

```

Comment:
It looks like these inversion functions are rotating the field 180deg (as opposed to just flipping X).
Is this intentional? (not saying it's wrong, just checking if we decided to change this since the in person conversation about this)

Reply:
Yeah that was intentional I was talking to Joe and Woodward about the plan and I think the worry was if you just flip x left and right rotations are inverted. But there may be other benefits im thinking of to just flipping x that im not thinking about.